lib/rucio/web/rest/flaskapi/v1/main.py

```diff
@@ -81,6 +81,30 @@ def apply_endpoints(app, modules):
 if not endpoints:
     endpoints = DEFAULT_ENDPOINTS
 
+    try:
+        endpoints_add = config_get('api', 'endpoints_add', raise_exception=False, default='')
+        endpoints_add = list(filter(bool, map(str.strip, endpoints_add.split(sep=','))))
+    except RuntimeError:
+        endpoints_add = []
+
+    try:
+        endpoints_remove = config_get('api', 'endpoints_remove', raise_exception=False, default='')
+        endpoints_remove = list(filter(bool, map(str.strip, endpoints_remove.split(sep=','))))
+    except RuntimeError:
+        endpoints_remove = []
```

Comment:
Please replace with:

```python
endpoints_add = config_get_list('api', 'endpoints_add', raise_exception=False, default=[])
endpoints_remove = config_get_list('api', 'endpoints_remove', raise_exception=False, default=[])
```

I have a commit sitting in a branch which replaces the existing code, which is not a good example to be replicated.